src/app/chip_data_model.gni

```diff
@@ -212,6 +211,7 @@ template("chip_data_model") {
         "${_app_root}/util/attribute-storage.cpp",
         "${_app_root}/util/attribute-table.cpp",
         "${_app_root}/util/ember-io-storage.cpp",
+        "${_app_root}/util/privilege-storage.cpp",
```

Comment:
Does dynamic server not use privilege bits from there?

Reply:
Thank you for your review. This was added for dynamic server, However, we don't need this anymore. so, I have reverted these changes